server/routes/exists.js

```diff
@@ -11,8 +11,11 @@ module.exports = async (req, res) => {
   }
 
   try {
-    await storage.exists(id);
-    res.sendStatus(200);
+    const meta = await storage.metadata(id);
+    res.set('WWW-Authenticate', `send-v1 ${meta.nonce}`);
+    res.send({
+      password: meta.pwd
```

Comment:
`meta.pwd` is a string iirc, '0' or '1'. It might be better to make the public type a boolean (or really anything besides '0' and '1') 😆

Reply:
oh dear that was a mess, string zero for false 😱 (I was ignorant about how redis stores values and assumed I would be retrieving an int, but I should have noticed the quotes in the output/json).